files/modules/cz/integsoft/keycloak/browser/authenticator/main/module.xml

```diff
@@ -12,19 +18,24 @@
         <module name="org.keycloak.keycloak-common"/>
         <module name="org.keycloak.keycloak-server-spi"/>
         <module name="org.keycloak.keycloak-server-spi-private"/>
-	<module name="org.keycloak.keycloak-model-infinispan"/>
+	    <module name="org.keycloak.keycloak-model-infinispan"/>

```

Comment:
suggestion(non-blocking): it looks like there is some stray indentation here and further down. might be worth using an XML formatter?

Reply:
Fixed. :)

Reply:
I still see stray indentation:
<img width="488" alt="Screenshot 2023-06-27 at 11 00 52 AM" src="https://github.com/mbta/keycloak-deploy/assets/214921/966e493b-6279-40e2-ace0-f9b6c1471845">